src/viur/core/bones/spam.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import random
+import typing as t
+from viur.core import i18n, current
+from viur.core.bones import NumericBone
+
+
+class SpamBone(NumericBone):
+    type = "numeric.spam"
+
+    def __init__(
+        self,
+        descr: str = i18n.translate(
+            "core.bones.spam.question",
+            "What is the result of the addition of {{a}} and {{b}}?"
+        ),
+        values: t.Iterable[str] = (
+            i18n.translate(f"core.bones.spam.value.{digit}", digit)
+            for digit in ("one", "two", "three", "four", "five", "six", "seven", "eight", "nine")
+        ),
+        required: bool = True,
+        precision: int = 0,
+        msg_invalid: str = i18n.translate(
+            "core.bones.spam.invalid",
+            "Your answer was wrong! Please try again."
+        ),
+        **kwargs
+    ):
+        if precision != 0:
+            raise ValueError(f"Cannot use {self.__class__.__name__!r} with a precision")
+
+        super().__init__(
+            required=required,
+            precision=precision,
```

Comment:
```suggestion
        msg_invalid: str = i18n.translate(
            "core.bones.spam.invalid",
            "Your answer was wrong! Please try again."
        ),
        **kwargs
    ):
        super().__init__(
            required=required,
            precision=0,
```

Reply:
Can you shortly explain what the change does?

Reply:
It removes the precision argument from signature since it can only be 0 anyway

Reply:
yes, but with your suggestion its possible to provide `precision` to the kwargs, and this will raise a TypeError because precision is given twice in this case. My initial version fixes this problem.

Reply:
Yes, but why is this a problem? Providing `precision=0` is redundant, bc it's already the default value and any other value would raise your `ValueError`.

Reply:
So you think, when assigning a `precision` to the SpamBone() an error `TypeError: super().__init__() got multiple values for keyword argument 'precision'` will be more helpful, than `ValueError: Cannot use 'SpamBone' with precision`?

Reply:
Not really, but the suggestion of my IDE to set the precision argument is not helpful too ![image](https://github.com/viur-framework/viur-core/assets/10272292/f74b94bc-aaaa-4721-8def-94e8c994ce5d)

What do u think about this?

```suggestion
        msg_invalid: str = i18n.translate(
            "core.bones.spam.invalid",
            "Your answer was wrong! Please try again."
        ),
        **kwargs
    ):
        if "precision" in kwargs:
            raise ValueError(f"Cannot use {self.__class__.__name__!r} with a precision")

        super().__init__(
            required=required,
            precision=0,
```